### DISCUSSION TITLE
Large Task List performance is slow.

### DISCUSSION THREAD

**Discussion by benarent** [community/community#4942] [Projects and Issues]
We've an issue with 378 tasks and it's running pretty slow. https://github.com/gravitational/teleport/issues/7572 I'm sure we are "Doing things wrong TM", but it would be great if you could speed it up a little for larger checklists.

**Comment by simonjur** [comment]
We're experiencing a similar issue at with a list long 505 tasks (in private repo) and one click there takes 5~6secs to update the list
